### PR TITLE
(spanish) README.md

### PR DESCRIPTION
**Pull request by hereje**
Translate file `README.md` to spanish

NOTE: This PR must be merged when the whole translation is completed, so that the links work properly.

### CODE REVIEW

File: translations/README.es.md [hunk 1]
```diff
@@ -8,122 +8,119 @@
...
 
-> **A note about quizzes**: All quizzes are contained [in this app](https://white-water-09ec41f0f.azurestaticapps.net/), for 52 total quizzes of three questions each. They are linked from within the lessons but the quiz app can be run locally; follow the instruction in the `quiz-app` folder.
+> **Una nota acerca de los exámenes**: Todos los exámenes están contenidos [en esta app](https://white-water-09ec41f0f.azurestaticapps.net/), por un total de 52 exámenes de 3 preguntas cada uno, Ellos están vinculados dentro de las lecciones pero la aplicación de exámenes puede ser ejecutada localmente; sigue las instrucciones en el directorio `quiz-app`.
```
Comment (Stevengez) [inline translations/README.es.md:80]:
```suggestion
> **Una nota acerca de los exámenes**: Todos los exámenes están contenidos [en esta app](https://white-water-09ec41f0f.azurestaticapps.net/), para un total de 52 exámenes de 3 preguntas cada uno, Ellos están vinculados dentro de las lecciones pero la aplicación de exámenes puede ser ejecutada localmente; sigue las instrucciones en el directorio `quiz-app`.
```

**Review by Stevengez — CHANGES_REQUESTED** [review]
Great translate overall, I added some suggestion to make it even better

**Review by abrildur — COMMENTED** [review]
¡Hola! Hice algunas modificaciones a las traducciones originales, espero que les sea útil :D

File: translations/README.es.md [hunk 2]
```diff
@@ -8,122 +8,119 @@
...
+> 🌍 Viaja alrededor del mundo mientras exploramos el aprendizaje automático a través de las culturas del mundo 🌍
 
-Azure Cloud Advocates at Microsoft are pleased to offer a 12-week, 26-lesson curriculum all about **Machine Learning**. In this curriculum, you will learn about what is sometimes called **classic machine learning**, using primarily Scikit-learn as a library and avoiding deep learning, which is covered in our forthcoming 'AI for Beginners' curriculum. Pair these lessons with our ['Data Science for Beginners' curriculum](https://aka.ms/datascience-beginners), as well!
+Los defensores de Azure Cloud en Microsoft están complacidos en ofrecer un plan de estudios de 12 semanas, con 26 lecciones, todas acerca de **aprendizaje automático**. En este plan de estudios, aprenderás acerca de lo que algunas veces es llamado **aprendizaje automático clásico**, usando principalmente Scikit-learn como una biblioteca y evitando el aprendizaje profundo, el cual se aborda en nuestro próximo plan de estudios 'IA para principiantes'. ¡También empareja estas lecciones con nuestro [plan de estudios 'Ciencia de Datos para principiantes'](https://aka.ms/datascience-beginners)!
```
[the remark this answers is not in the document]
Comment (hereje) [reply translations/README.es.md:16]: Done!

File: translations/README.es.md [hunk 3]
```diff
@@ -8,122 +8,119 @@
...
 
-Travel with us around the world as we apply these classic techniques to data from many areas of the world. Each lesson includes pre- and post-lesson quizzes, written instructions to complete the lesson, a solution, an assignment, and more. Our project-based pedagogy allows you to learn while building, a proven way for new skills to 'stick'.
+Viaja con nosotros alrededor del mundo mientras aplicamos estas técnicas clásicas a los datos de distintas áreas del mundo. Cada lección incluye exámenes previos y posteriores a la lección, instrucciones escritas para completar la lección, una solución, una asignación y más. Nuestra pedagogía basa en proyectos te permite aprender mientras construyes, una forma probada para que las nuevas habilidades se te 'peguen'.
```
[the remark this answers is not in the document]
Comment (hereje) [reply translations/README.es.md:25]: Done!

**Comment by hereje** [comment]
@abrildur , gracias por el aporte.

**Review by jlooper — APPROVED** [review]

**Pull request merged**